.travis.yml

```diff
@@ -47,6 +47,11 @@ addons:
             - python3-gst-1.0
             - unrar
 
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu xenial main"
+    - sudo apt-get -y update
+    - sudo apt-get install sqlite3 -y
```

Comment:
Great, I understand this turned out to be much larger than expected (mostly because of Query related changes) but I think the result is worth it :)